install/Dockerfile.faiss

```diff
@@ -1,9 +1,9 @@
 FROM ann-benchmarks
 
-RUN apt-get update && apt-get install -y libopenblas-base libopenblas-dev libpython-dev python-numpy python-pip
+RUN apt-get update && apt-get install -y libopenblas-base libopenblas-dev libpython-dev python-numpy python-pip python-dev
 RUN git clone https://github.com/facebookresearch/faiss lib-faiss
-RUN cd lib-faiss && cp example_makefiles/makefile.inc.Linux makefile.inc && make -j4 py BLASLDFLAGS=/usr/lib/x86_64-linux-gnu/libopenblas.so.0
-ENV PYTHONPATH lib-faiss
+RUN export PYTHON=python && cd lib-faiss && cp example_makefiles/makefile.inc.Linux makefile.inc && ./configure && make -j4 BLASLDFLAGS=/usr/lib/x86_64-linux-gnu/libopenblas.so.0  && make py && make install
+ENV PYTHONPATH lib-faiss/python
```

Comment:
thanks for fixing this, have been meaning to!

Reply:
Sure. It seems that they also support Python3 by now, but it needs some manual changes to the example makefile. Probably some `sed` magic would do the trick, but since it's running in docker it's not big deal anyways.